samples/performance/image_compression_control/image_compression_control.cpp

```diff
@@ -152,13 +152,13 @@ void ImageCompressionControlSample::create_render_context()
 	else
 	{
 		// Filter default list to those formats that support compression
+		auto const                     &default_surface_priority_list = get_surface_priority_list();
 		std::vector<VkSurfaceFormatKHR> new_surface_priority_list;
-
-		for (size_t i = 0; i < get_surface_priority_list().size(); i++)
+		for (size_t i = 0; i < default_surface_priority_list.size(); i++)
```

Comment:
Maybe an even nicer formulation would be
`for ( auto const & surface_priority : get_surface_priority_list() )`